lib/web_ui/dev/environment.dart

```diff
@@ -135,6 +135,12 @@ class Environment {
     'dev',
   ));
 
+  /// Path to the "test" directory containing web engine tests.
```

Comment:
puff, how come we don't have it until now :D this is useful!

Reply:
I know, was surprised myself.